src/config.rs

```diff
@@ -56,6 +56,7 @@ const POOL_UNUSED_RELEASE_TIMEOUT: &str = "POOL_UNUSED_RELEASE_TIMEOUT";
 const CONNECTION_TERMINATION_DEADLINE: &str = "CONNECTION_TERMINATION_DEADLINE";
 const ENABLE_ORIG_SRC: &str = "ENABLE_ORIG_SRC";
 const PROXY_CONFIG: &str = "PROXY_CONFIG";
+const IPV6_DISABLED: &str = "IPV6_DISABLED";
```

Comment:
To keep the semantics similar with https://github.com/istio/istio/blob/master/manifests/charts/istio-cni/values.yaml#L55 (and the rest of Istio), I'd prefer to invert this.

These two settings will 90% of the time be kept in lockstep, and having one be (disable) and the other be (enable) is needlessly confusing.

Reply:
(if we want to make both `disable` that's fine too - but they should be consistent, and changed together)